spec/controllers/tasks_controller_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'webmock/rspec'
```

Comment:
This import shouldn't be required here (haven't run the code), since it is already loaded through `rails_helper`.